### PR TITLE
fix: extra line in plain html with CRLF

### LINKED ISSUE #665: If plain html code is provided, empty lines are added
I have alot of php files that contain pure html code.

Prettier PHP takes those files and adds plain empty html lines between every line.

Before:
```php
<section class="[ c-social-foot ]  [ has-text-centered ]">
  <div class="[ container  is-fluid ]">
    <div class="[ columns  is-centered ]">
      <div class="[ column ]">
```

After
```php
<section class="[ c-social-foot ]  [ has-text-centered ]">

  <div class="[ container  is-fluid ]">

    <div class="[ columns  is-centered ]">

      <div class="[ column ]">
```

This is very odd, it should remain as it was.

### PR DESCRIPTION
**Pull request by evilebottnawi**
fixes #665

### CODE REVIEW

File: tests_config/run_spec.js [hunk 1]
```diff
@@ -27,7 +27,11 @@ function run_spec(dirname, parsers, options) {
       filename[0] !== "." &&
       filename !== "jsfmt.spec.js"
     ) {
-      const source = read(path).replace(/\r\n/g, "\n");
+      let source = read(path);
+
+      if (!options.keepEOL) {
```
Comment (evilebottnawi) [inline tests_config/run_spec.js]: New option for test to allow keep original EOL

**Comment by evilebottnawi** [comment]
Feel free to feedback

**Comment by vielhuber** [comment]
- When will there be a release?

- How can I test this? When I run ````npm update --global prettier/plugin-php```, I still have 0.7.0.

**Comment by evilebottnawi** [comment]
@vielhuber Plan on today:
1. More test with CRLF and fix problems
2. Add appveyor CI
3. Release 0.8.0

**Comment by czosel** [comment]
Is a new option really the way to go here? How is Prettier for JS handling files with windows line endings?

**Comment by evilebottnawi** [comment]
@czosel prettier for js doesn't support html + php, also it is not public option, it is option only for tests

**Comment by evilebottnawi** [comment]
I.e. you can't set this option, just to allow testing CRLF and LF

**Comment by czosel** [comment]
Ah ok, if the option is not public then it's fine :+1:

**Pull request merged**